communications.md

```diff
@@ -1,68 +1,89 @@
 # Communications
 
-## Opt-in
+## General guidelines
 
-Opt-in to Slack channels, calendars, or Asana projects, etc., that you are interested in or obligated to. Configure notifications on these tools to suit this mindset:
+Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. Therefore, the focus should be on real problem solving and creative endeavors instead of drowning oneself in processes and tools when communicating with one another.
+
+## First line of Communication: Texting
+
+We use a few tools to help team members communicate with one another: Slack, Gmail and Asana. Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, use Slack, Gmail or Asana to contact another team member asynchronously (in an intermittent manner).
```

Comment:
If we generalize email and calendar under one banner, `G Suite` may be more accurate than just Gmail